src/freenet/support/io/NativeThread.java

```diff
@@ -10,7 +10,8 @@
 /**
  * Do *NOT* forget to call super.run() if you extend it!
  * 
- * @see http://archives.freenetproject.org/thread/20080214.235159.6deed539.en.html
+ * @see "Old link - http://archives.freenetproject.org/thread/20080214.235159.6deed539.en.html"
```

Comment:
I left it there as i wasnt 100% that i have listed the correct new link.
You want it removed?